corehq/apps/smsforms/models.py

```diff
@@ -163,6 +163,15 @@ def status(self):
             else:
                 return ugettext_noop('Not Finished')
 
+    @property
+    def status_api(self):
+        if self.submission_id:
+            return "complete"
+        elif self.session_is_open and self.session_type == XFORMS_SESSION_SMS:
+            return "in-progress"
+        else:
+            return "not-finished"
```

Comment:
I'm not entirely sure. This is just duplicating what the `status` property does. I've refactored to remove the duplicate logic and to use constants: 649ac383e09

Reply:
Following up again re `not-finished` vs `in-progress`, it appears this only applies to IVR events. It's been like this since it was added: https://github.com/dimagi/commcare-hq/commit/b5dae782fedcc9509421631669cf7f1d958e4741#diff-cd82d34a4d32a57355cf795d95d2a964b44b1919ac118e093c168d36209db36aR35

I'll ask Giovanni if he recalls: https://github.com/dimagi/commcare-hq/commit/b5dae782fedcc9509421631669cf7f1d958e4741#r51919687